tools/packaging/packaging.psm1

```diff
@@ -105,8 +105,9 @@ function Start-PSPackage {
         $actualParams = @()
 
         $crossGenCorrect = $false
-        if ($Runtime -match "arm") {
-            # crossgen doesn't support arm32/64
+        if ($Runtime -match "arm" -or $Type -eq 'min-size') {
+            ## crossgen doesn't support arm32/64;
```

Comment:
I wonder if it is still not supported.

Reply:
I'm not sure. This is the existing code.

Reply:
This can be filed as a separate issue and addressed there.